scripts/upgrade.sh.in

```diff
@@ -169,3 +178,4 @@ run "Upgrading bootstrap" upgrade_bootstrap
 run "Setting cluster version to $DESTINATION_VERSION" patch_kubesystem_namespace
 run "Launching the pre-upgrade" launch_pre_upgrade
 run "Launching the upgrade" launch_upgrade
+run "Launching the post-upgrade cleanup" launch_post_upgrade
```

Comment:
```suggestion
run "Launching the post-upgrade" launch_post_upgrade
```

Do not specify cleanup please we may want in the future to do some other actions